lib/galaxy/config/sample/file_sources_conf.yml.sample

```diff
@@ -235,6 +235,6 @@
   id: onedata1
   label: Onedata
   doc: Your Onedata files - configure an access token via user preferences
-  accessToken: ${user.preferences['onedata|access_token']}
-  onezoneDomain: ${user.preferences['onedata|onezone_domain']}
-  disableTlsCertificateValidation: ${user.preferences['onedata|disable_tls_certificate_validation']}
+  access_token: ${user.preferences['onedata|access_token']}
+  onezone_domain: ${user.preferences['onedata|onezone_domain']}
```

Comment:
może przy tych preferencjach usera byśmy dopisali "Minimal supported Onezone version is 21.02.4."